sphinx/source/docs/user_guide/interaction/callbacks.rst

```diff
@@ -201,7 +201,7 @@ CustomJS for Hover
 ''''''''''''''''''
 
 The ``HoverTool`` has a callback which comes with two pieces of built-in data: the
-`index`, and the `geometry`. The `index` is the indices of any points that the
+`index` and the `geometry`. The `index` is the indices of any points that the
```

Comment:
unrelated to your changes, but I think these should be double backticks for sphinx to do anything with them

Reply:
I was wondering about that too. With a single backtick, Sphinx will render those words in italics, which does seem odd. I have changed them to double backticks for the next commit.